debian/copyright

```diff
@@ -1,23 +1,54 @@
-Source: https://go.googlecode.com/hg/
+Format: http://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: golang
+Source: https://github.com/golang/go
 
 Files: *
 Copyright: © 2009, 2010, The Go Authors.  All rights reserved.
 License: Go
 
-Files: include/*, src/lib9/*, src/libbio/*, src/pkg/runtime/386/*,
- src/libmach/*, src/pkg/runtime/arm/vlop.s, src/pkg/runtime/arm/vlrt.c,
- src/pkg/runtime/arm/memset.s, src/pkg/runtime/arm/memmove.s,
- src/pkg/runtime/amd64/memmove.s, src/pkg/runtime/amd64/memmove.s,
- src/pkg/math/fltasm_amd64.s, src/cmd/6c/txt.c, src/cmd/gopack/ar.c,
- src/cmd/ld/lib.[ch], src/cmd/6a/a.[yh], src/cmd/6a/lex.c,
- src/cmd/5g/gobj.c, src/cmd/5g/list.c, src/cmd/5g/gsubr.c,
- src/cmd/5g/opt.h, src/cmd/8c/*, src/cmd/goyacc/goyacc.go,
- src/cmd/5l/*, src/cmd/6g/reg.c, src/cmd/6g/gobj.c,
- src/cmd/6g/peep.c, src/cmd/6g/list.c, src/cmd/6g/gsubr.c,
- src/cmd/6g/opt.c, src/cmd/8l/*, src/cmd/nm/nm.c,
- src/cmd/gc/bits.c, src/cmd/cc/*, src/cmd/5a/*, src/cmd/8g/reg.c,
- src/cmd/8g/gobj.c, src/cmd/8g/peep.c, src/cmd/8g/list.c,
- src/cmd/8g/gsubr.c, src/cmd/8g/opt.h, src/cmd/5c/*, src/cmd/6l/*,
+Files: include/*
+ src/lib9/*
```

Comment:
So yeah, this looks better, but it's still clearly not up to date for 1.5: lib9 (and lots and lots of other files) do not exist any more. *Surely* there's some tool can tell us paths that are not covered by d/copyright and paths in d/copyright that no longer exist?